### ISSUE TITLE
UX suggestion: improvement to publishing dialog in umbraco 8

### ISSUE THREAD

**Issue by shearer3000** [umbraco/Umbraco-CMS#7900]
Hi

We have an umbraco 8 site using variants with 10+ languages. When the content editor is publishing a page the “Ready to Publish?” dialog presents every available language to them even if they only made changes to 1 (the typical case), so it becomes a bit overwhelming.

Is it possible for this dialog to only show the languages that have pending changes? If a full language list is still required for other reasons/use cases then maybe the content editor could click a button to reveal these (but they are hidden by default).

In summary, the point just being that the majority use case is that the content editor typically only needs to see the languages with pending changes to publish.

Thanks

Andrew

![ready-to-publish](https://user-images.githubusercontent.com/2433680/78509520-94252d00-77e2-11ea-85a1-50acb032e815.png)

**Comment by nielslyngsoe** [comment]
Hi @shearer3000 

Thanks for bringing up an issue like this. User Experience issues are very important but not seen that often.

I already had similar thoughts some time back, https://github.com/umbraco/Umbraco-CMS/issues/7611
Which has already been developed and if I'm not mistaken it will be part of next release, must be 8.7 then.

Thanks for your input and please let us know how the new version works for you.

And please bring more of these types of problems, THANKS!

**Comment by nielslyngsoe** [comment]
This will be fixed in 8.7.0 when branch https://github.com/umbraco/Umbraco-CMS/pull/7499 has been merged.
